packages/datatrak-web/src/views/NotFoundPage.tsx

```diff
@@ -3,12 +3,22 @@
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 import React from 'react';
+import { DialogActions } from '@material-ui/core';
+import { Button, ErrorDisplay } from '../components';
 
-export const NotFoundPage = () => {
+interface NotFoundPageProps {
+  error?: Error;
+  title?: string;
+}
+
+export const NotFoundPage = ({ error, title = '404: Page not found' }: NotFoundPageProps) => {
```

Comment:
Can I make a request to update the NotFoundPage to be called ErrorPage since it's now generic. For example we might use it for Permission Denied errors or something. Also would you be able to tweak the styles so the Paper is in the middle of the page 🙏